.github/workflows/release.yaml

```diff
@@ -116,4 +116,50 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           GORELEASER_CURRENT_TAG: ${{ github.event.inputs.version }}
-          GORELEASER_PREVIOUS_TAG: ${{ github.event.inputs.pre_version }}
\ No newline at end of file
+          GORELEASER_PREVIOUS_TAG: ${{ github.event.inputs.pre_version }}
+
+  helm_release:
+    name: Publish Helm Chart
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Replace release version
+        id: replace_version
+        run: find . -type f -name "values.yaml" -o -name "Chart.yaml" |
+          xargs sed -i -e s/latest/${{ github.event.inputs.version }}/g
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.4.0
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.1.0
+
+      - name: Add Bitnami Repository
+        run: helm repo add bitnami https://charts.bitnami.com/bitnami
+
+      - name: Run chart-testing (lint)
+        run: ct lint --check-version-increment=false --validate-maintainers=false
+          --target-branch ${{ github.event.inputs.version }}
```

Comment:
I think we need `Run chart-testing` also in PR workflow - we don't want to fail on the lint only in the release - is it make sense?

Reply:
Probably validating against master is ok, no ? (So removing the target-branch option)

Reply:
I mean, we need to run this step in the the CI workflow (`.github/workflows/ci.yml`) to verify that we don't break the chart in any PR.
I just want to avoid failing during release.